src/components/Trade/Trade.tsx

```diff
@@ -19,10 +17,27 @@ export const Trade = ({ defaultBuyAssetId }: TradeProps) => {
   const location = useLocation()
   const [hasSetDefaultValues, setHasSetDefaultValues] = useState<boolean>(false)
 
-  const { dispatch: swapperDispatch } = useSwapperState()
-
   const methods = useForm({ mode: 'onChange' })
 
+  const updateFiatBuyAmount = useSwapperStore(state => state.updateBuyAmountFiat)
+  const updateFiatSellAmount = useSwapperStore(state => state.updateSellAmountFiat)
+  const updateSellAssetFiatRate = useSwapperStore(state => state.updateSellAssetFiatRate)
+  const updateBuyAssetFiatRate = useSwapperStore(state => state.updateBuyAssetFiatRate)
+  const updateFeeAssetFiatRate = useSwapperStore(state => state.updateFeeAssetFiatRate)
+  const updateAction = useSwapperStore(state => state.updateAction)
+  const updateIsExactAllowance = useSwapperStore(state => state.updateIsExactAllowance)
+  const updateAmount = useSwapperStore(state => state.updateAmount)
+  const updateQuote = useSwapperStore(state => state.updateQuote)
+  const updateTrade = useSwapperStore(state => state.updateTrade)
+  const updateBuyAsset = useSwapperStore(state => state.updateBuyAsset)
+  const updateSellAsset = useSwapperStore(state => state.updateSellAsset)
+  const updateBuyAmountCryptoPrecision = useSwapperStore(
+    state => state.updateBuyAmountCryptoPrecision,
+  )
+  const updateSellAmountCryptoPrecision = useSwapperStore(
+    state => state.updateSellAmountCryptoPrecision,
+  )
+
```

Comment:
This may keep more tidy since some of these are re-defined
https://docs.pmnd.rs/zustand/guides/auto-generating-selectors#create-the-following-function:-createselectors